utils_linux.go

```diff
@@ -454,3 +462,14 @@ func startContainer(context *cli.Context, spec *specs.Spec, action CtAct, criuOp
 	}
 	return r.run(spec.Process)
 }
+
+/* setLinuxPersonality sets the Linux execution personality. For more information see the personality syscall documentation.
+   checkout getLinuxPersonalityFromStr() from libcontainer/specconv/spec_linux.go for type conversion.
+*/
+func setLinuxPersonality(persona int) error {
+	_, _, errno := syscall.Syscall(syscall.SYS_PERSONALITY, uintptr(persona), 0, 0)
+	if errno != 0 {
+		return fmt.Errorf("syscall: SYS_PERSONALITY failed with error code: %d", errno)
```

Comment:
please wrap errno using `%w` (so it could be unwrapped).

Reply:
@kolyshkin this is resolved in latest commit.